Spectre.playground/Sources/Playground.swift

```diff
@@ -1,13 +1,19 @@
 class PlaygroundReporter : ContextReporter {
   var depth = 0
 
-  func print(message:String) {
+#if swift(>=3.0)
+  func print(_ message: String) {
+    let indentation = String(repeating: " ", count: depth * 2)
+      Swift.print("\(indentation)\(message)")
+  }
+#else
```

Comment:
Thanks for the pull request.

I'm not sure Spectre still build on Swift < 3 so I think we can remove this conditional (here and below). Or does it still work?